{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -0,0 +1,224 @@
+# Please refer to data parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+if env == "prod":
+    # TODO(required) : set data for prod workspace
+    data = None
+elif env == "staging":
+    # TODO(required) : set data for staging workspace
+    data = None
+else:
+    raise Exception("Unknown environment. Please select 'prod' or 'staging' as environment name")
+
+# The string name of a column from data that contains evaluation labels.
+# Call get_targets_from_recipe() to get targets from recipe configs if mlflow recipe is used. 
+# Please refer to targets parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(required) : targets
+targets = get_targets_from_recipe()
+
+
+# A string describing the model type. The model type can be either "regressor" and "classifier".
+# Call get_model_type_from_recipe() to get model type from recipe configs if mlflow recipe is used. 
+# Please refer to model_type parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(required) : model_type
+model_type = get_model_type_from_recipe()
+
+# Custom metrics to be included. Set it to None if custom metrics are not needed.
+# Please refer to custom_metrics parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(optional) : define custom metric function as necessary
+# TODO(optional) : custom_metrics
+custom_metrics = []
+
+# Define model validation rules. Set it to None if validation rules are not needed.
+# Please refer to custom_metrics parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(optional) : validation_thresholds
+validation_thresholds = {}
+
+# A dictionary of additional configurations to supply to the evaluator.
+# Please refer to evaluator_config parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
+# TODO(optional) : evaluator_config
+evaluator_config = {}
+
+# COMMAND ----------
+
+client.get_model_version(model_name, model_version).description
+
+# COMMAND ----------
+
+eval_result = None
+err = None
+
+def log_to_model_description(run, success):
+    run_info = run.info
+    run_link = "#mlflow/experiments/{0}/runs/{1}".format(run_info.experiment_id, run_info.run_id)
+    description = client.get_model_version(model_name, model_version).description
+    if success:
+        status = "succeeded"
+        emoji = ":white_check_mark:"
+    else:
+        status = "failed"
+        emoji = ":x:"
+    if description != "":
+        description += """
+            ---
+        """.replace(" ", "")
+    description += "### {0}Model validation {1}. Refer to [model validation run]({2}) for details\n".format(emoji, status, run_link)
```

Comment:
Prefer more succinct log: `Model validation status: {0}\nValidation Details: {1}`

Reply:
Before
<img width="1064" alt="Screen Shot 2023-01-31 at 8 57 45 PM" src="https://user-images.githubusercontent.com/12734110/215955308-5eb13bea-855b-417d-b733-f54da8111d71.png">

after
<img width="1027" alt="Screen Shot 2023-01-31 at 8 57 29 PM" src="https://user-images.githubusercontent.com/12734110/215955330-713d6897-63de-4aef-87d1-3c208cf09261.png">

Reply:
@jinzhang21  Please let me know if it looks good.